### PR TITLE
Updated create api workflow

### PR DESCRIPTION
**Pull request by arlemi**
Updated the create new API flow documentation from the Build view on the app. 

This is meant to go live on the 9th of March.

### CODE REVIEW

File: src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ You can connect various components of your API development and testing process t
 
 To access the API Builder, open __APIs__ from the left sidebar in the Postman app. You can open and edit any existing APIs from here—Postman will automatically open the most recent version of an API by default.
 
-<img alt="Create API" src="https://assets.postman.com/postman-docs/create-new-api.jpg" width="250px"/>
+<img alt="Create API" src="https://user-images.githubusercontent.com/5029719/76172321-69689a00-618c-11ea-8dfd-2bd6e89ef6e2.png" width="250px"/>
```
Comment (tristandenyer) [inline src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md]: Why are we not uploading our images to S3? GitHub is not serving images optimally, and we want to serve them through our CDN.

I uploaded your image to S3. Please use this URL: https://assets.postman.com/postman-docs/create-new-api-v7.png

File: src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md [hunk 2]
```diff
@@ -50,13 +50,13 @@ You can connect various components of your API development and testing process t
...
 
 Click __Create an API__ or __+ New API__—_you will need to be signed into your Postman account_.
 
-![New API](https://assets.postman.com/postman-docs/untitled-api-created.jpg)
+![New API](https://user-images.githubusercontent.com/5029719/76172306-4b029e80-618c-11ea-9de8-0e3a4b7cd5db.png)
```
Comment (tristandenyer) [inline src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md]: I uploaded your image to S3 and optimized it. Please use this URL: https://assets.postman.com/postman-docs/untitled-api-created-v7.jpg

File: src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ You can connect various components of your API development and testing process t
...
-<img alt="Create API" src="https://assets.postman.com/postman-docs/create-new-api.jpg" width="250px"/>
+<img alt="Create API" src="https://assets.postman.com/postman-docs/create-new-api-v7.png" width="250px"/>
 
 Click __Create an API__ or __+ New API__—_you will need to be signed into your Postman account_.
 
-![New API](https://assets.postman.com/postman-docs/untitled-api-created.jpg)
+![New API](https://assets.postman.com/postman-docs/untitled-api-created-v7.jpg)
 
-Enter a name for your API—you can rename, delete, or remove it from the workspace using the __View more actions__ (__...__) menu in the left sidebar.
+Enter a name and a version, then select a schema type and format for your API. You can also import an API schema directly from that modal. The API can be renamed, deleted, or removed from the workspace using the __View more actions__ (__...__) menu in the left sidebar.
```
Comment (SueSmith) [inline src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md]: For the second sentence I'd maybe just go for something like "You can optionally import an API schema directly at this stage." Then for the last sentence if we can have "You can rename, delete, or remove the API from the workspace..." just to get rid of passive that would be great.

**Comment by arlemi** [comment]
@SueSmith Thanks for the suggestions, I've updated it.

**Pull request merged**